cfgov/unprocessed/css/molecules/global-banner.less

```diff
@@ -38,7 +38,9 @@
     }
 
     .o-expandable_target {
-        .btn__secondary();
+        // Taken from .a-btn__secondary.
```

Comment:
Same question about duplication here. What's stopping us from including the `secondary` class in the markup?